### DISCUSSION TITLE
侧边菜单栏能支持一下树型结构吗？

### DISCUSSION THREAD

**Discussion by wangjiancheng180** [element-plus/element-plus#7266] [General]
自己用递归试了下，发现效果很差，用递归折叠菜单动画会显得卡，效果太差了

**Comment by wangjiancheng180** [comment]
我好像解决了<el-menu>里的<el-sub-menu>标签不能有其他标签包裹，包裹之后折叠动画就会卡顿

**Comment by msidolphin** [comment]
卡顿情况请提供下demo，Menu本身是支持树形结构的啊

**Reply by wangjiancheng180** [reply]
这个卡顿的情况好像是我缓存的问题已经解决了。其实我想表达的是侧边栏menu能不想像选择器那样传入数据源，自动可以展开形成树型结构
